roles/config/cluster/base/templates/configs/kerberos-6.x-7.x.j2

```diff
@@ -30,6 +30,3 @@ SPARK3_ON_YARN:
   GATEWAY:
     spark_io_encryption_enabled: true
     spark_network_encryption_enabled: true
-STREAMS_MESSAGING_MANAGER:
```

Comment:
Is this a side effect -- seems out-of-scope for the PR...

Reply:
No a side effect. It's a change to remove deprecated parameters. It's in a separate commit, but I forgot to update the PR title and description to include them all.